packages/corvid-cli/src/commands/create-rc.js

```diff
@@ -0,0 +1,78 @@
+const _ = require("lodash");
+const chalk = require("chalk");
+const base64 = require("../utils/base64");
+const createSpinner = require("../utils/spinner");
+const { login } = require("./login");
+const getMessage = require("../messages");
+const fetch = require("node-fetch");
+const commandWithDefaults = require("../utils/commandWithDefaults");
+const { UserError } = require("corvid-local-logger");
+const { readCorvidConfig } = require("../utils/corvid-config");
+const fs = require("fs");
+
+const basePublicRcUrl =
+  "https://editor.wix.com/html/editor/web/api/publish-rc/";
+
+const parseErrorMessage = errorMessage => {
+  if (_.includes(errorMessage, "RC already exists"))
+    return getMessage("CreateRc_Command_Already_Exists");
+
+  if (_.includes(errorMessage, "Session not found"))
+    return getMessage("CreateRc_Login_Command_Failure");
+
+  return `${getMessage("CreateRc_Command_Failure")}: ${errorMessage}`;
+};
+
+const buildFetchUrl = async () => {
+  const config = await readCorvidConfig(process.cwd());
+  const file = fs.readFileSync("./src/assets/site/siteInfo.wix", {
+    encoding: "utf8"
+  });
+  const decodedContent = base64.decode(JSON.parse(file).content);
```

Comment:
Get the siteId from the clone process / some other API (instead of decoding the .wix file)